LibreNMS/Data/Store/Rrd.php

```diff
@@ -384,6 +384,8 @@ private function command($command, $filename, $options)
         if ($command == 'last' && $this->init(false)) {
             // send this to our synchronous process so output is guaranteed
             $output = $this->sync_process->sendCommand($cmd);
+        } elseif ($command == 'list') {
```

Comment:
This should be combined with the above if statement.
```php
if (in_array($command, ['last', 'list']) && $this->init(false)) {
```
The second statement makes sure the sync process is started.